// cards/cards.js
import dispel from '../../assets/cards/dispel.jpg';
import feveredVisions from '../../assets/cards/fevered-visions.jpg';
import hinder from '../../assets/cards/hinder.jpg';
import island from '../../assets/cards/island.jpg';
import lordOfTheUnreal from '../../assets/cards/lord-of-the-unreal.jpg';
import manaLeak from '../../assets/cards/mana-leak.jpg';
import negate from '../../assets/cards/negate.jpg';
import plains from '../../assets/cards/plains.jpg';
import ponder from '../../assets/cards/ponder.jpg';
import removeSoul from '../../assets/cards/remove-soul.jpg';
import thoughtseize from '../../assets/cards/thoughtseize.jpg';

import dispelThumbnail from '../../assets/cards/thumbnails/dispel.gif';
import feveredVisionsThumbnail from '../../assets/cards/thumbnails/fevered-visions.gif';
import hinderThumbnail from '../../assets/cards/thumbnails/hinder.gif';
import islandThumbnail from '../../assets/cards/thumbnails/island.gif';
import lordOfTheUnrealThumbnail from '../../assets/cards/thumbnails/lord-of-the-unreal.gif';
import manaLeakThumbnail from '../../assets/cards/thumbnails/mana-leak.gif';
import negateThumbnail from '../../assets/cards/thumbnails/negate.gif';
import plainsThumbnail from '../../assets/cards/thumbnails/plains.gif';
import ponderThumbnail from '../../assets/cards/thumbnails/ponder.gif';
import removeSoulThumbnail from '../../assets/cards/thumbnails/remove-soul.gif';
import thoughtseizeThumbnail from '../../assets/cards/thumbnails/thoughtseize.gif';

export default [

  // TODO: Knight of the Reliquary
  // TODO: Knight of the Reliquary
  // TODO: Knight of the Reliquary
  // TODO: Knight of the Reliquary
  // TODO: Tireless Tracker
  // TODO: Force of Will
  // TODO: Negate (second)
  // TODO: Psionic Blast

  {
    date: '2017 Feb. 28',
    description: 'This piece was commissioned as part of a 5-piece commission set.',
    image: ponder,
    price: 30,
    set: 'Magic Player Rewards',
    sold: true,
    thumbnail: ponderThumbnail,
    title: 'Ponder',
    url: 'ponder'
  },

  {
    date: '2017 Feb. 27',
    description: 'This piece was commissioned as part of a 5-piece commission set.',
    image: hinder,
    price: 30,
    set: 'Magic Player Rewards',
    sold: true,
    thumbnail: hinderThumbnail,
    title: 'Hinder',
    url: 'hinder'
  },

  {
    date: '2017 Feb. 18',
    description: 'This piece was commissioned as part of a 5-piece commission set.',
    image: manaLeak,
    price: 30,
    set: 'Magic Player Rewards',
    sold: true,
    thumbnail: manaLeakThumbnail,
    title: 'Mana Leak',
    url: 'mana-leak'
  },

  {
    date: '2017 Feb. 16',
    description: 'This piece was commissioned as part of a 5-piece commission set.',
    image: removeSoul,
    price: 30,
    set: 'Magic Player Rewards',
    sold: true,
    thumbnail: removeSoulThumbnail,
    title: 'Remove Soul',
    url: 'remove-soul'
  },

  {
    date: '2017 Feb. 15',
    description: 'This piece was commissioned as part of a 5-piece commission set.',
    image: negate,
    price: 30,
    set: 'Magic Player Rewards',
    sold: true,
    thumbnail: negateThumbnail,
    title: 'Negate',
    url: 'negate'
  },

  {
    date: '2017 Feb. 10',
    image: thoughtseize,
    price: 30,
    set: 'THS-107',
    sold: true,
    thumbnail: thoughtseizeThumbnail,
    title: 'Thoughtseize',
    url: 'thoughtseize'
  },

  {
    date: '2017 Feb. 1',
    image: feveredVisions,
    price: 15,
    set: 'SOI-244',
    sold: true,
    thumbnail: feveredVisionsThumbnail,
    title: 'Fevered Visions',
    url: 'fevered-visions'
  },

  {
    date: '2017 Jan. 25',
    image: lordOfTheUnreal,
    price: 45,
    set: 'M12-062',
    sold: true,
    thumbnail: lordOfTheUnrealThumbnail,
    title: 'Lord of the Unreal',
    url: 'lord-of-the-unreal'
  },

  {
    date: '2016 Jul. 1',
    image: dispel,
    price: null,
    set: 'BFZ-076',
    sold: true,
    thumbnail: dispelThumbnail,
    title: 'Dispel',
    url: 'dispel'
  },

  {
    date: '2016 Jun. 1',
    image: island,
    price: 45,
    set: 'SOI-286',
    sold: true,
    thumbnail: islandThumbnail,
    title: 'Island',
    url: 'island'
  },

  {
    date: '2016 May. 1',
    image: plains,
    price: 35,
    set: 'M14-230',
    sold: true,
    thumbnail: plainsThumbnail,
    title: 'Plains',
    url: 'plains'
  }
];
