// cards/index.js
import dispel from '../../assets/cards/thumbnails/dispel.gif';
import feveredVisions from '../../assets/cards/thumbnails/fevered-visions.gif';
import hinder from '../../assets/cards/thumbnails/hinder.gif';
import island from '../../assets/cards/thumbnails/island.gif';
import lordOfTheUnreal from '../../assets/cards/thumbnails/lord-of-the-unreal.gif';
import manaLeak from '../../assets/cards/thumbnails/mana-leak.gif';
import negate from '../../assets/cards/thumbnails/negate.gif';
import plains from '../../assets/cards/thumbnails/plains.gif';
import ponder from '../../assets/cards/thumbnails/ponder.gif';
import removeSoul from '../../assets/cards/thumbnails/remove-soul.gif';
import thoughtseize from '../../assets/cards/thumbnails/thoughtseize.gif';

export default [

  // TODO: Knight of the Reliquary
  // TODO: Knight of the Reliquary
  // TODO: Knight of the Reliquary
  // TODO: Knight of the Reliquary
  // TODO: Tireless Track
  // TODO: Force of Will

  {
    date: '2017 Feb. 28',
    description: 'This piece was commissioned with a &lquo;friend rate&rquo; as part of a 5-piece commission set.',
    image: ponder,
    price: 10,
    sold: true,
    title: 'Ponder',
    url: 'ponder'
  },

  {
    date: '2017 Feb. 27',
    description: 'This piece was commissioned with a &lquo;friend rate&rquo; as part of a 5-piece commission set.',
    image: hinder,
    price: 10,
    set: 'Magic Player Rewards',
    sold: true,
    title: 'Hinder',
    url: 'hinder'
  },

  {
    date: '2017 Feb. 18',
    description: 'This piece was commissioned with a &lquo;friend rate&rquo; as part of a 5-piece commission set.',
    image: manaLeak,
    price: 10,
    set: 'Magic Player Rewards',
    sold: true,
    title: 'Mana Leak',
    url: 'mana-leak'
  },

  {
    date: '2017 Feb. 16',
    description: 'This piece was commissioned with a &lquo;friend rate&rquo; as part of a 5-piece commission set.',
    image: removeSoul,
    price: 10,
    set: 'Magic Player Rewards',
    sold: true,
    title: 'Remove Soul',
    url: 'remove-soul'
  },

  {
    date: '2017 Feb. 15',
    description: 'This piece was commissioned with a &lquo;friend rate&rquo; as part of a 5-piece commission set.',
    image: negate,
    price: 20,
    set: 'Magic Player Rewards',
    sold: true,
    title: 'Negate',
    url: 'negate'
  },

  {
    date: '2017 Feb. 10',
    image: thoughtseize,
    price: 30,
    set: 'THS-107',
    sold: true,
    title: 'Thoughtseize',
    url: 'thoughtseize'
  },

  {
    date: '2017 Feb. 1',
    image: feveredVisions,
    price: 15,
    set: 'SOI-244',
    sold: true,
    title: 'Fevered Visions',
    url: 'fevered-visions'
  },

  {
    date: '2017 Jan. 25',
    image: lordOfTheUnreal,
    price: 45,
    set: 'M12-062',
    sold: true,
    title: 'Lord of the Unreal',
    url: 'lord-of-the-unreal'
  },

  {
    date: '2016 Jul. 1',
    image: dispel,
    price: null,
    set: 'BFZ-076',
    sold: true,
    title: 'Dispel',
    url: 'dispel'
  },

  {
    date: '2016 Jun. 1',
    image: island,
    price: 45,
    set: 'SOI-286',
    sold: true,
    title: 'Island',
    url: 'island'
  },

  {
    date: '2016 May. 1',
    image: plains,
    price: 35,
    set: 'M14-230',
    sold: true,
    title: 'Plains',
    url: 'plains'
  }
];
